src/lib/swapper/swappers/ThorchainSwapper/utils/getThresholdedAffiliateBps/getThresholdedAffiliateBps.ts

```diff
@@ -0,0 +1,67 @@
+import { getConfig } from 'config'
+import type { Asset } from 'lib/asset-service'
+import { bn, convertPrecision } from 'lib/bignumber/bignumber'
+import { convertBasisPointsToDecimalPercentage } from 'state/slices/tradeQuoteSlice/utils'
+
+import { THORCHAIN_OUTBOUND_FEE_RUNE_THOR_UNIT } from '../../constants'
+import type { MidgardPoolResponse } from '../../types'
+import { THORCHAIN_FIXED_PRECISION } from '../constants'
+import { assetIdToPoolAssetId } from '../poolAssetHelpers/poolAssetHelpers'
+import { thorService } from '../thorService'
+
+export const getOutboundFeeInSellAssetThorBaseUnit = (assetPricePrecision: string) => {
```

Comment:
I get it we took this vernacular from the THOR pool endpoint, but I find price to be very confusing and convey `market data fiat value` intent.

What about `runeRatio` or perhaps the vernacular that's used across investor packages, i.e `pricePerShare`, which could be made a clearer `runePerShare` or `runePerAsset` here since this is a single-purpose RUNE per asset ratio?

Reply:
`runePerAsset` is a great improvement, ill update to use that!